### PR TITLE
Allow query epochs to be specified as list or string

### LINKED ISSUE #326: Update Query to allow epochs to be specified as list or string
Per title.

### PR DESCRIPTION
**Pull request by ddobie**
Fix #326.

### CODE REVIEW

File: docs/components/query.md [hunk 1]
```diff
@@ -170,7 +170,7 @@ All matches within the `crossmatch_radius` are returned instead of just the clos
 These `Query` keyword arguments control the data that is used to perform the query or the data returned.
 
 **`epochs`**  
-A comma-separated list of epochs (entered as a string) to search.
+A list of epochs or a comma-separated string of epochs to search.
```
Comment (ajstewart) [inline docs/components/query.md:173]: So that it's really clear, perhaps tab the example right below this bit to show the same entries using both methods.

Comment (ddobie) [reply docs/components/query.md:173]: How about that?

File: docs/components/query.md [hunk 2]
```diff
@@ -170,14 +170,18 @@ All matches within the `crossmatch_radius` are returned instead of just the clos
...
 
-!!! example
+!!! example "Example: Querying with a comma-separated string"
     ```python
     my_query = Query(..., epochs="1,2,6x,8,9")
     ```
+!!! example "Example: Querying with a list"
+    ```python
+    my_query = Query(..., epochs=["1","2","6x","8","9"])
+    ```
```
Comment (ajstewart) [inline docs/components/query.md:184]:
```suggestion
!!! example "Example: Defining the epochs parameter"
    === "Comma-separated string"
        ```python
        my_query = Query(..., epochs="1,2,6x,8,9")
        ```
    === "List"
        ```python
        my_query = Query(..., epochs=["1","2","6x","8","9"])
        ```
```
Personally I'd go for something like this, but perhaps an extra click to actually see it is in fact a bad thing 😅

Comment (ddobie) [reply docs/components/query.md:184]: Ah no, I like that!

**Review by ajstewart — APPROVED** [review]

**Pull request merged**